openshift-ci/build-root/OWNERS

```diff
@@ -1,7 +1,9 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-- mohammedzee1000
+- anandrkskd
+- feloy
 
 reviewers:
-- mohammedzee1000
+- anandrkskd
+- feloy
```

Comment:
@anandrkskd @feloy
I am unsure about this one, but you both seem to have done some work on this front, so I added you both as owners. Let me know if I should change it.